docs/src/tutorials/mathprogbase.md

```diff
@@ -0,0 +1,97 @@
+# Transitioning from MathProgBase
+
+MathOptInterface is a replacement for [MathProgBase.jl](https://github.com/JuliaOpt/MathProgBase.jl).
+However, it is not a direct replacement.
+
+## Transitioning a solver interface
+
+Writing a solver interface in MathOptInterface is more work than the equivalent
```

Comment:
I wouldn't say this. It may potentially be much closer to the solver. MPB was tailored to 3 specific format. If you didn't fit then it was way more work. CSDP MPB wrapper was so complicated I had to create SemidefiniteModels.jl. Now it's quite simple.

Reply:
For conic solver, it will be much simpler than with MPB with MatrixOfConstraints.

Reply:
Make a PR with the suggested wording. I didn't think it was controversial that it was more complicated! We now have lots of constraint types and attributes and copy-to and caching and layers. It might be simpler for you, but it certainly isn't for most people.

Reply:
It depends what you mean by simpler I guess :)